src/dotnet/CarbonAware.Plugins.JsonReaderPlugin/CarbonAwareJsonReaderPlugin.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System.Collections;
+using CarbonAware.Model;
+using Microsoft.Extensions.Logging;
+using System.Reflection;
+using Newtonsoft.Json;
+
+namespace CarbonAware.Plugins.JsonReaderPlugin;
+
+public class CarbonAwareJsonReaderPlugin : ICarbonAware
```

Comment:
Documentation strings seem to be missing (use /// <inheritdoc /> to inherit fromt eh interface for public methods.